go.mod

```diff
@@ -21,13 +21,13 @@ require (
 	github.com/containerd/errdefs v0.1.0
 	github.com/containerd/fifo v1.1.0
 	github.com/containerd/go-cni v1.1.10
-	github.com/containerd/imgcrypt v1.2.0-rc1.0.20240709223013-f3769dc3e47f
+	github.com/containerd/imgcrypt v1.2.0-rc1.0.20240724013606-c091455481f6
 	github.com/containerd/log v0.1.0
 	github.com/containerd/nydus-snapshotter v0.14.1-0.20240806063146-8fa319bfe9c5
 	github.com/containerd/platforms v0.2.1
-	github.com/containerd/stargz-snapshotter v0.15.2-0.20240709063920-1dac5ef89319
-	github.com/containerd/stargz-snapshotter/estargz v0.15.2-0.20240709063920-1dac5ef89319
-	github.com/containerd/stargz-snapshotter/ipfs v0.15.2-0.20240709063920-1dac5ef89319
+	github.com/containerd/stargz-snapshotter v0.15.2-0.20240826180748-fbc3f6a1d4aa
```

Comment:
avoid updating hash for unrelated things.

Reply:
These changes occurred after running `go mod tidy`, which I needed to run after adding the `gopsutil` package. Let me see how I can work around this

Reply:
probably is fine

Reply:
Tried it, and `go mod tidy` did not up these dependencies.
Maybe you did `go get -u` somehow?
Anyhow, I don't think we should touch these at this time.

Reply:
fixed in latest push